.gitignore

```diff
@@ -17,6 +17,8 @@
 *.pdf
 *.tif
 *.h5
+*.hdf5
+*.all
```

Comment:
Not sure I understand what `*all` is ignoring.

Reply:
was ignoring the 'gfxxyy.all' files for the Kurucz linelists when they were being downloaded to the repo folder - no longer applicable after integrating with Common API.